translate/cloud-client/src/main/java/com/example/translate/HybridGlossaries.java

```diff
@@ -0,0 +1,301 @@
+
+      Feature feature = Feature.newBuilder().setType(Type.DOCUMENT_TEXT_DETECTION).build();
+
+      AnnotateImageRequest request =
+              AnnotateImageRequest.newBuilder()
+                      .addFeatures(feature)
+                      .setImage(image)
+                      .build();
+
+      requests.add(request);
+
+      BatchAnnotateImagesResponse response = client.batchAnnotateImages(requests);
+      List<AnnotateImageResponse> responses = response.getResponsesList();
+
+      StringBuilder result = new StringBuilder();
+
+      for (AnnotateImageResponse res : responses) {
+        result.append(String.format("%s\n", res.getFullTextAnnotation().getText()));
+      }
+
+      return result.toString();
+    } catch (IOException e) {
+      return String.format("Failed to get the text from the image.\n%s\n", e.getMessage());
+    }
+  }
+  // [END translate_hybrid_vision]
+
+  // [START translate_hybrid_create_glossary]
+  /** Creates a GCP glossary resource
+   * Assumes you've already manually uploaded a glossary to Cloud Storage
+   *
+   * ARGS
+   * @param languages list of languages in the glossary
+   * @param projectId GCP project id
+   * @param glossaryDisplayName name you want to give this glossary resource
+   * @param glossaryUri the uri of the glossary you uploaded to Cloud Storage
+   * 
+   **/
+  static void createGlossary(List<String> languages, String projectId,
+                             String glossaryDisplayName, String glossaryUri) {
+    try (TranslationServiceClient translationServiceClient = TranslationServiceClient.create()) {
+
+      LocationName locationName =
+              LocationName.newBuilder().setProject(projectId).setLocation("us-central1").build();
+
+
+      LanguageCodesSet languageCodesSet =
+              LanguageCodesSet.newBuilder().addAllLanguageCodes(languages).build();
+
+      GcsSource gcsSource = GcsSource.newBuilder().setInputUri(glossaryUri).build();
+
+      GlossaryInputConfig glossaryInputConfig =
+              GlossaryInputConfig.newBuilder().setGcsSource(gcsSource).build();
+
+      GlossaryName glossaryName =
+              GlossaryName.newBuilder()
+                      .setProject(projectId)
+                      .setLocation("us-central1")
+                      .setGlossary(glossaryDisplayName)
+                      .build();
+
+      Glossary glossary =
+              Glossary.newBuilder()
+                      .setLanguageCodesSet(languageCodesSet)
+                      .setInputConfig(glossaryInputConfig)
+                      .setName(glossaryName.toString())
+                      .build();
+
+      CreateGlossaryRequest request =
+              CreateGlossaryRequest.newBuilder()
+                      .setParent(locationName.toString())
+                      .setGlossary(glossary)
+                      .build();
+
+      // Call the API
+      Glossary response = translationServiceClient.createGlossaryAsync(request)
```

Comment:
nit: This looks like it might be a style violation. Can you make sure Checkstyle is running?

Something like this might be better: 

```java
Glossary response = translationServiceClient
              .createGlossaryAsync(request)
              .get(300, TimeUnit.SECONDS);
```